woocommerce-gateway-lightning.php

```diff
@@ -116,6 +123,7 @@ public function process_payment( $order_id ) {
             'currency'    => $order->get_currency(),
             'amount'      => $order->get_total(),
             'description' => self::make_desc($order),
+            'expiry'      => $this->get_option('invoice_expiry'),
```

Comment:
Would be better if this also specified the default value of 2 hours, so it'll be used as the default without having to go through the admin page and hitting 'save'. This could matter for users that upgraded the plugin from a previous version, or that enable the plugin programmatically and not through the gui.

`get_option()` accepts a a default value as its second argument.

Reply:
Good idea. Updated!